_data/sites.json

```diff
@@ -1519,6 +1519,18 @@
             "bohemia.net"
         ]
     },
+    
+        {
+        "name": "Bol.com",
+        "url": "https://www.bol.com/nl/rnwy/account/privacy/delete",
+        "difficulty": "hard",
+        "notes": "Need to request deletion on a page within the account settings. Confirm after via link in email, which often doesn't work, because you need to be logged in to the account for the confirmation link to be working. This is not clear.",
```

Comment:
I'm a bit confused. When you click the link to request deletion, do you get an e-mail instantly to confirm?

Or do you need to mail them to request proper deletion?

Reply:
No you get an email after a few days with a link to confirm your request.

Reply:
If the link doesn't work you can email them, but you have to check it manually it seems.

Reply:
Ah ok, so I suggest we remove the e-mail field. We add it here when people have to mail the support to request for deletion.

Reply:
Well you do sometimes as the links do not always work. I speak from experience having had to delete about 10 accounts with them.